dozer-sql/Cargo.toml

```diff
@@ -29,3 +30,4 @@ tokio = { version = "1", features = ["rt", "macros"] }
 [features]
 python = ["dozer-sql-expression/python"]
 onnx = ["dozer-sql-expression/onnx"]
+wasm = ["wasmtime"]
```

Comment:
Looks like we should remove `wasmtime` dependency and change it to `dozer-sql-expression/wasm`?